pkgs/development/python-modules/django-json-widget/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  pythonOlder,
+  setuptools,
+  python,
+  pytestCheckHook,
+  pytest-django,
+}:
+
+buildPythonPackage rec {
+  pname = "django-json-widget";
+  version = "2.0.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "jmrivas86";
+    repo = "django-json-widget";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-GY6rYY//n8kkWCJZk6OY+EOBv62ocNovNmE/ai8VCn4=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ django ];
+
+  nativeCheckInputs = [
+    pytest-django
+    pytestCheckHook
+  ];
+
+  preCheck = ''
+    export DJANGO_SETTINGS_MODULE=tests.settings
+  '';
+
+  pythonImportCheck = [ "django_json_widget" ];
+
+  meta = {
+    description = "Alternative widget that makes it easy to edit the jsonfield field of django";
+    homepage = "https://github.com/jmrivas86/django-json-widget";
+    changelog = "https://github.com/jmrivas86/django-json-widget/blob/v${version}/CHANGELOG.rst";
+    license = lib.licenses.mit;
```

Comment:
Can you clarify the license with upstream?

https://github.com/jmrivas86/django-json-widget/blame/a111908cf2ce7410ba65e0d69bee442d6681e5a5/setup.py#L59
https://github.com/jmrivas86/django-json-widget/blame/a111908cf2ce7410ba65e0d69bee442d6681e5a5/setup.py#L69
https://github.com/jmrivas86/django-json-widget/blob/a111908cf2ce7410ba65e0d69bee442d6681e5a5/LICENSE#L2

Reply:
Added a note and a reference to upstream issue https://github.com/jmrivas86/django-json-widget/issues/93

Thanks for pointing that out!

Reply:
According to upstream it's MIT only, but leaving the reference until its fixed